docs/Manual/04 Fusing.md

```diff
@@ -116,55 +116,57 @@ for i in range(0, 16, 4):
+Accera guarantees the preservation of the fused schedules *for each value of the fused dimensions*, regardless of how the fused schedule is transformed. For example, in the example above, the fused dimensions are `i` and `j`. Therefore, for any concrete value of `i` and `j`, the corresponding operation from `schedule0` is guaranteed to execute before the corresponding operation from `schedule1`, regardless of how the fused schedule is transformed. More specifically, for each `i` and `j`, the operation `C[i, j] += A[i, j]` is guaranteed to execute before the operation `C[i, j] *= B[i, j]`, no matter how we transform the fused schedule. Since those are the only operations that interact with `C[i,j]`, the Accera guarantee is sufficient, and we can claim that the fused schedule is safe. With this assurance, the programmer can apply any sequence of transformations without worrying about the correctness of the resulting implementation.
 
-However, note that not every fusing operation creates a safe schedule. For example, imagine that we had fused `schedule0` and `schedule1` differently:
+However, not every fusing operation creates a safe schedule. For example, consider a scenario where we fused `schedule0` and `schedule1` differently:
 ```python
 # Reorder schedule1 before fusing
 schedule1.reorder(j1, i1)
 # Fuse schedule0 with the reordered schedule1
 schedule_t = acc.fuse(schedule0, schedule1)
 f, a, b = schedule_t.get_indices()
 ```
-In this unnatural example, `i0` and `j1` are fused and named `a`, and `i1` and `j0` are fused and named `b`. As before, Accera guarantees that for each value of `a` and `b` the operation `C[a, b] += A[a, b]` is executed before `C[b, a] *= B[b, a]`. As noted above, the fusing operation itself preserves logical equivalence, but if we proceed to transform the fused schedule as follows,
+In this unusual example, `i0` and `j1` are fused and named `a`. Similarly,`i1` and `j0` are fused and named `b`. As mentioned above, Accera guarantees that, for each value of `a` and `b`, the operation `C[a, b] += A[a, b]` is executed before `C[b, a] *= B[b, a]`. The fusing operation itself preserves the logical equivalence. However, the underlying logic is changed if we transform the fused schedule as follow: 
```

Comment:
"unnatural" is correct. It's more than unusual (it's rather contrived to demonstrate a possible error situation)